documentation/master-openshift.adoc

```diff
@@ -24,6 +24,8 @@ include::assemblies/assembly-service-admin-guide.adoc[leveloffset=+1]
 
 include::assemblies/assembly-tenant-guide.adoc[leveloffset=+1]
 
+include::assemblies/assembly-iot-guide.adoc[leveloffset=+1]
```

Comment:
Make sure you add this to master-kubernetes.adoc as well.

Reply:
Will do, when it make it work :)